src/plugins/intellisense/index.tsx

```diff
@@ -404,6 +404,24 @@ export default class Intellisense extends PluginController {
     });
   };
 
+  // allows mathquill inputs that only allow arithmetic to selectively disable intellisense
+  isActiveElementValidForIntellisense() {
```

Comment:
I think this whole function can be replaced with `document.activeElement.closest(".yes-intellisense.no-intellisense.dcg-settings-view-container").classList.contains("yes-intellisense") ?? true`

Reply:
Oh smart! I'll change it.

Reply:
Adding commas seemed to do the trick!